demos/python_demos/monitors_extension/monitors_extension.cpp

```diff
@@ -0,0 +1,228 @@
+// Copyright (C) 2018-2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#define PY_SSIZE_T_CLEAN
+#include <Python.h>
+
+#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
+#include "numpy/arrayobject.h"
+
+#include <exception>
+#include <monitors/presenter.h>
+
+struct PresenterObject {
+    PyObject_HEAD
+    std::unique_ptr<Presenter> _presenter;
```

Comment:
This struct should not have members with constructors/destructors in it, because objects of it will be constructed/destroyed by C code, and thus the constructor/destructor will never be invoked.

Reply:
Fixed